packages/addon-dev/src/rollup-public-entrypoints.ts

```diff
@@ -12,10 +13,28 @@ export default function publicEntrypoints(args: {
 }): Plugin {
   return {
     name: 'addon-modules',
-    buildStart() {
-      for (let name of walkSync(args.srcDir, {
-        globs: args.include,
-      })) {
+    async buildStart() {
+      // Include *.hbs to add template-only files to publicEntrypoints
+      let matches = walkSync(args.srcDir, {
+        globs: [...args.include, 'components/**/*.hbs'],
```

Comment:
This seems to be forcing every template-only component in `components/**/*.hbs` to be exposed a public entrypoint? We wouldn't want that.

I don't think we should be adding anything to the user's pattern. If the users wants to match template-only components they can give us a pattern that includes .hbs. We can make that the default in the example rollup config.

I would also prefer not to give the path "components/" any special behavior. We can treat every `.hbs` file as a component. Apps have other scenarios they need to worry about like route templates, but we don't ,and even if some addon author needs that they can customize their config.

Reply:
removed!